jax/api.py

```diff
@@ -769,13 +756,13 @@ def pmap(fun, axis_name=None, devices=None, backend=None, axis_size=None):
       Optional, a string representing the xla backend. 'cpu', 'gpu', or 'tpu'.
 
   Returns:
-    A parallelized version of ``fun`` with arguments that correspond to those of
-    ``fun`` but each with an additional leading array axis (with equal sizes)
-    and with output that has an additional leading array axis (with the same
-    size).
+    A parallelized version of ``fun`` with arguments that correspond to
+    those of ``fun``, but with extra array axes at positions indicated by
```

Comment:
Perhaps this can be clarified: "...but each with an additional array axis at positions..."